mask_the_face.py

```diff
@@ -133,6 +133,7 @@
                 tqdm.write(str_p)
 
             split_path = f.rsplit(".")
+            split_path = ["".join(split_path[:-1]),split_path[-1]]
```

Comment:
```suggestion
            split_path = [".".join(split_path[:-1]),split_path[-1]]
```

Dot is needed to properly join the string again